hack/setup-kind-in-ci.sh

```diff
@@ -99,26 +99,12 @@ EOF
 infomsg "Create Kind LoadBalancer via MetalLB"
 lb_addr_range="255.70-255.84"
 
-kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/namespace.yaml
-kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/metallb.yaml
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.13.5/config/manifests/metallb-native.yaml
```

Comment:
Do you know if `v0.13.5` also works with kube 1.18? Looks like that's the oldest version we test against from line 60.

Reply:
I'll test

Reply:
Setting 
ISTIO_VERSION="1.7.0"
TARGET_BRANCH="v1.24"
Even in master, I'm receiving an error: 

[kubelet-check] It seems like the kubelet isn't running or healthy.
[kubelet-check] The HTTP call equal to 'curl -sSL http://localhost:10248/healthz' failed with error: Get http://localhost:10248/healthz: dial tcp [::1]:10248: connect: connection refused.

Maybe any other step that I'm missing?

Reply:
Ah actually we don't run these tests anyways against v1.24 since they were not backported: https://github.com/kiali/kiali/tree/v1.24.9. Sorry about that. Can you test with 1.36 then and remove the v1.24 conditionals for kube/istio version?

Reply:
Tested with: 

TARGET_BRANCH="v1.36"
ISTIO_VERSION="1.10.0"

And it is working fine. 

I had to do a change, though, for both master and the branch: 
 helm-charts/_output/charts/kiali-server-*.tgz

For some reason was not getting the SNAPSHOT in the name. But probably this is just in my environment.

Reply:
> kube 1.18

That seems awfully old.  I think kube 1.21 (the version in OpenShift 4.8) would be fine.